pkg/kv/kvserver/node_liveness.go

```diff
@@ -128,6 +128,8 @@ type LivenessMetrics struct {
 	HeartbeatLatency   *metric.Histogram
 }
 
+// TODO(irfansharif): We should/could cut out a pkg/nodeliveness here.
```

Comment:
If you think doing this would make you really happy, then I would support doing it now. Otherwise, and this is my intuition about what's best right now, remove the comment and leave as is. You're not materially changing liveness here, and we know liveness is a clear candidate to move to a pkg.

Can you spin this commit out into a separate PR? This can then be landed already, once you've added a simple unit test.

Reply:
re: pkg/liveness, https://github.com/cockroachdb/cockroach/pull/56221. I'll send a follow-on PR for GetLivenessFromKV.

Reply:
> I'll send a follow-on PR for GetLivenessesFromKV.

Done: https://github.com/cockroachdb/cockroach/pull/56243.